argo/common/run_completion.go

```diff
@@ -27,7 +27,7 @@ type RunCompletionEvent struct {
 	RunName               *NamespacedName `json:"runName,omitempty"`
 	RunId                 string          `json:"runId"`
 	ServingModelArtifacts []Artifact      `json:"servingModelArtifacts"`
-	Artifacts             []Artifact      `json:"artifacts,omitempty"`
+	Artifacts             []Artifact      `json:"artifacts"`
```

Comment:
`Artifacts` is now always populated as an empty list, so no need for the `omitempty`